shuttering/prod/apps-hmcts-net.yml

```diff
@@ -63,9 +63,9 @@ cname:
 
     # Juror
   - name: "juror"
-    shutter: false
+    shutter: true
```

Comment:
Setting this true will shutter all the juror production apps, is this intentional?  do we have Change Request?

Reply:
Yes this is intentional, this juror app is not the production version.